ims/ostis_tech/lib_ostis/sectn_lib_reusable_comp_kpm/reusable_sc_agents/lib_scp_agents/activity_automatisation/classification_agents/lib_component_agent_of_finding_all_drafts_of_author/agent_of_finding_all_drafts_of_author.scsi

```diff
@@ -0,0 +1,317 @@
+agent_of_finding_all_drafts_of_author
+=> nrel_main_idtf:
+	[агентная scp-программа поиска всех черновиков разработчикa] (* <- lang_ru;; *);
+	[agent scp-program of finding all drafts of author] (* <- lang_en;; *);
+	<- agent_scp_program;;
+
+scp_program -> agent_of_finding_all_drafts_of_author
+	(*
+	-> rrel_params: .agent_of_finding_all_drafts_of_author_params 
```

Comment:
No need to name  .agent_of_finding_all_drafts_of_author_params, use ...

Reply:
Updates were done